src/garage/np/exploration_policies/epsilon_greedy_policy.py

```diff
@@ -48,6 +48,15 @@ def __init__(self,
         self._total_env_steps = 0
         self._last_total_env_steps = 0
 
+    @property
+    def epsilon(self):
+        """Float: the instantaneous level of exploration noise."""
+        return self._epsilon
+
+    @epsilon.setter
```

Comment:
I doubt this can set the epsilon, as the epsilon here is inferred by `self._total_env_steps`. So I recommend to set the total_env_steps or use `get_param_values` and `set_param_values`.

Reply:
Ah you're right, thanks! I'll fix that.